akka-http-cors/src/test/scala/ch/megard/akka/http/cors/CorsDirectivesSpec.scala

```diff
@@ -68,6 +68,21 @@ class CorsDirectivesSpec extends WordSpec with Matchers with Directives with Sca
       }
     }
 
+    "accept pre-flight requests with a null Origin" in {
+      val settings = CorsSettings.defaultSettings
+      Options() ~> Origin(Seq.empty) ~> `Access-Control-Request-Method`(GET) ~> {
+        route(settings)
+      } ~> check {
+        status shouldBe StatusCodes.OK
+        response.headers should contain theSameElementsAs Seq(
+          `Access-Control-Allow-Origin`.`null`,
+          `Access-Control-Allow-Methods`(settings.allowedMethods),
+          `Access-Control-Max-Age`(1800),
+          `Access-Control-Allow-Credentials`(true)
+        )
+      }
+    }
+
```

Comment:
Could you add a test checking that when `allowed-origins` is set to something else than `*`, the `Origin: null` should be rejected.

Reply:
Added